num/src/bigint.rs

```diff
@@ -830,6 +831,18 @@ macro_rules! construct_bigint {
             }
         }
 
+        impl ::core::str::FromStr for $name {
+            #[inline]
+            fn from_str(s: &str) -> Result<Self, Self::Err> {
```

Comment:
Not sure I understand exactly what the code below does, so test case will be appreciated.

My understanding is that for FromStr implementation we support only decimals (like standard library) and should have `from_str_radix` for other bases (which may be another PR, not necessary doing it here). Second, what you need is not a map over split iterator but a map over `chunks` iterator...

Reply:
ok, i'm seeing